### PR TITLE
CRUD groups

### PR DESCRIPTION
**Pull request by leplatrem**
* [x] URL / forms / cabling
* [x] Write some tests
* [x] Type flow stuff

### CODE REVIEW

**Comment by n1k0** [comment]
First review pass done, it just looks great. Kudos!

**Comment by leplatrem** [comment]
Ready to merge/review :) `/me goes to grab lunch`

File: test/sagas/bucket_test.js [hunk 1]
```diff
@@ -18,6 +19,12 @@ const collectionData = {
   displayFields: [],
 };
 
+const groupData = {
+  id: "group",
+  members: ["tartenpion", "account:abc"]
```
Comment (n1k0) [inline test/sagas/bucket_test.js]: major nit: *tartempion* :p

File: scripts/components/CollectionHistory.js [hunk 2]
```diff
@@ -52,7 +52,7 @@ class HistoryRow extends Component {
   }
 }
 
-function HistoryTable(props) {
+export function HistoryTable(props) {
```
Comment (n1k0) [inline scripts/components/CollectionHistory.js]: This is certainly generic enough to live in its own module.

Comment (leplatrem) [inline scripts/components/CollectionHistory.js]: We did it in the other pull-request

**Comment by n1k0** [comment]
I've now read this huge piece of diff twice and couldn't spot any major mistake, and running the patch locally works and provides the expected features :)

The only bit I'm slightly uncomfortable with is the new loadRoute saga signature but couldn't find any better way without a big scary refactoring involved, so I think we can live with it for now.

r+

**Pull request merged**